2022.md

```diff
@@ -2,6 +2,7 @@
 
 ## July (1)
```

Comment:
Let's update the blog count

Reply:
Done! https://github.com/wso2/iam-blogs/pull/149/commits/2435f427c14ac108cae2cb2f1d60f4d94cdc864a